stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -19,14 +19,14 @@
 /// It's not named just Glibc so that it doesn't conflict in the event of a
 /// future official glibc modulemap.
 module SwiftGlibc [system] {
-% if CMAKE_SDK in ["LINUX", "FREEBSD", "CYGWIN"]:
+% if CMAKE_SDK in ["LINUX", "FREEBSD", "OPENBSD", "CYGWIN"]:
```

Comment:
Might be nicer to split out the file separately for FreeBSD's libc.

Reply:
To clarify: you mean like how there's a separate modulemap for bionic, but still called SwiftGlibc?

Reply:
Yeah, exactly so.

Reply:
Sounds good. I'll pull the modulemap changes into a separate commit (in this PR).